src/reducers/dataReducer.js

```diff
@@ -57,6 +59,41 @@ const initialDataState = {
 
 module.exports = (state = initialDataState, action) => {
   switch (action.type) {
+    case 'DATA_CENTERS_FETCH_REQUESTED': {
+      return { ...state, isLoadingDataCenterFacilities: true };
+    }
+
+    case 'DATA_CENTERS_FETCH_SUCCEEDED': {
+      return {
+        ...state,
+        isLoadingDataCenterFacilities: false,
+        dataCenterFacilities: Boolean(action) && Array.isArray(action.payload) ?
+          action.payload.map(dataCenterFacility => (
+            Boolean(dataCenterFacility) && {
+              ...dataCenterFacility,
+              consentToOpenData: dataCenterFacility.consent_to_open_data,
+              createdAt: dataCenterFacility.created_at,
+              energyInputRestEndpoint: dataCenterFacility.energy_input_rest_endpoint,
+              energyInputStreamEndpoint: dataCenterFacility.energy_input_stream_endpoint,
+              energyInputStreamTopic: dataCenterFacility.energy_input_stream_topic,
+              energyOutputRestEndpoint: dataCenterFacility.energy_output_rest_endpoint,
+              energyOutputStreamEndpoint: dataCenterFacility.energy_output_stream_endpoint,
+              energyOutputStreamTopic: dataCenterFacility.energy_output_stream_topic,
+              equipmentInventoryRestEndpoint: dataCenterFacility.equipment_inventory_rest_endpoint,
+              lat: dataCenterFacility.geo_lat,
+              lng: dataCenterFacility.geo_lon,
```

Comment:
Please avoid abbreviations (specially if the notation is different from the one that comes from the API), transform the variables names only when using them on the APIs that require specific notations